### ISSUE TITLE
Create tags and categories for the blog

### ISSUE THREAD

**Issue by shannonbux** [gatsbyjs/gatsby#4640]
## Description

As the Gatsbyjs.org blog grows, it'd be nice and probably eventually essential to have tags and categories. Suggestions on how to get this accomplished?

**Comment by KyleAMathews** [comment]
It's a fairly easy thing to add on design/programming — we even have a tutorial :-) https://www.gatsbyjs.org/docs/adding-tags-and-categories-to-blog-posts/

I think the main thing we need to do at this point is to come up with a starter list of categories / tags.

**Comment by shannonbux** [comment]
by "starter list" do you mean initial list? Or categories/tags for starters? @KyleAMathews

**Comment by KyleAMathews** [comment]
Yeah, initial list of tags/categories — e.g. look through our blog posts right now and decide what's a minimal taxonomy for describing the content there + ones we'll probably have in the future. It doesn't have to be perfect as they're easy to change.

**Comment by fk** [comment]
I just started reviewing the UX and Documentation project todos and stumbled upon this issue.
I'm happy to report that we have tags/categories for the blog since #4782 thanks to @calcsam who also added an initial set of tags for most of the blog posts.

All current tags can be seen at https://www.gatsbyjs.org/blog/tags. This page and Sam's tagging efforts give us a head start in refining the list of tags—something which I won't do right now. ;-)

But let me add two quick thoughts that I had:

* Let's make https://www.gatsbyjs.org/blog/tags more accessible to our readers/users — ATM the page can only be reached by clicking one of the tags on a blog post page and then following the "All tags" link. We could
  * add the whole list of tags to the blog index page
  * just link the page there
  * go for a combination of listing the ~10 most "popular" tags (as in "number of posts tagged") along with a "Browse all tags" link
  * curate a list of tags, again followed by a "Browse all tags" link

* Seeing
  * https://www.gatsbyjs.org/blog/2018-05-24-launching-new-gatsby-company/ and
  * https://www.gatsbyjs.org/blog/2018-04-06-introducing-gatsby-rfc-process/
  * https://www.gatsbyjs.org/blog/2018-02-28-documentation-project/
  * https://www.gatsbyjs.org/blog/2017-12-20-introducing-the-gatsby-ux-research-program/
  * https://www.gatsbyjs.org/blog/gatsby-v1/
  * https://www.gatsbyjs.org/blog/gatsby-first-beta-release/
  and anticipating a new article announcing v2 (beta) made me think about a new tag along the lines of "Official Announcements". Does that make sense to y'all?
